packages/react-widgets/src/DateTimePickerInput.js

```diff
@@ -62,16 +64,16 @@ class DateTimePickerInput extends React.Component {
   }
 
   render() {
-    let { disabled, readOnly } = this.props
+    let { className = '', disabled, readOnly, type = 'text' } = this.props
     let { textValue } = this.state
 
     let props = Props.omitOwn(this)
 
     return (
       <Input
         {...props}
-        type="text"
-        className="rw-widget-input"
+        type={type}
```

Comment:
i don't actually think this should be configurable, any other input type is likely to break the inputs and trigger conflicting browser behavior.

Reply:
Good point 👍